models/clan_test.go

```diff
@@ -1087,22 +1087,22 @@ var _ = Describe("Clan Model", func() {
 			It("Should return clan by search term", func() {
 				err := testing.CreateClanNameTextIndexInMongo(GetTestMongo, player.GameID)
 				Expect(err).NotTo(HaveOccurred())
-				clans, err := SearchClan(testDb, testMongo, player.GameID, "SEARCH", 10)
+				clans, err := SearchClan(testDb, testMongo, player.GameID, "SEARCH", 10, true)
```

Comment:
Try to avoid raw boolean values on function calls. Adding an enumeration is easier to understand.

Reply:
Will do.

Reply:
Done.